It seems that it is not yet working. I get errors on all basic gtfs tests python -m tools.test_req ./tools/tool_scripts/meta_test.txt res.xlsx 

example: 

```

=============================================================
[7/9] Running: python -m tools.script_runner ./tools/tool_scripts/script_regression_basic.txt mylog.log nl4gtfs
Returned 1.



-------------------------------------------------------------


2026-01-22 13:44:40,313 - script_runner - INFO - nl4gtfs - step: 1: clean_tmp C:/Users/ue71603/Downloads/nl4gtfs
2026-01-22 13:44:40,317 - script_runner - INFO - Command 'clean_tmp' executed for folder: C:/Users/ue71603/Downloads/nl4gtfs

2026-01-22 13:44:40,318 - script_runner - INFO - nl4gtfs - step: 2: download_input_file C:/Users/ue71603/Downloads/nl4gtfs
2026-01-22 13:44:40,318 - script_runner - INFO - File exists already. Will use: C:/Users/ue71603/Downloads/nl4gtfs\NeTEx_WSF_WSF_20241203_20241203.xml.gz
filepaths: C:/Users/ue71603/Downloads/nl4gtfs\NeTEx_WSF_WSF_20241203_20241203.xml.gz
2026-01-22 13:44:40,318 - script_runner - INFO - Command 'download_input_file' executed for url: https://github.com/user-attachments/files/18202171/NeTEx_WSF_WSF_20241203_20241203.xml.gz

2026-01-22 13:44:40,318 - script_runner - INFO - nl4gtfs - step: 3: conv.netex_to_db [C:/Users/ue71603/Downloads/nl4gtfs\NeTEx_WSF_WSF_20241203_20241203.xml.gz] C:/Users/ue71603/Downloads/nl4gtfs/03.mdbx
the arguments [['C:/Users/ue71603/Downloads/nl4gtfs\\NeTEx_WSF_WSF_20241203_20241203.xml.gz'], 'C:/Users/ue71603/Downloads/nl4gtfs/03.mdbx']
2026-01-22 13:44:54,479 - script_runner - INFO - Execution time: 14.1 seconds for nl4gtfs - step: 3: conv.netex_to_db [C:/Users/ue71603/Downloads/nl4gtfs\NeTEx_WSF_WSF_20241203_20241203.xml.gz] C:/Users/ue71603/Downloads/nl4gtfs/03.mdbx

2026-01-22 13:44:54,479 - script_runner - INFO - nl4gtfs - step: 4: conv.gtfs_db_to_db C:/Users/ue71603/Downloads/nl4gtfs/03.mdbx C:/Users/ue71603/Downloads/nl4gtfs/04.mdbx
the arguments ['C:/Users/ue71603/Downloads/nl4gtfs/03.mdbx', 'C:/Users/ue71603/Downloads/nl4gtfs/04.mdbx']
gtfs_operator_line_memory
2026-01-22 13:44:54,736 - script_runner - INFO - Processing ServiceJourneys
2026-01-22 13:44:54,808 - script_runner - INFO - Processing Calendars
2026-01-22 13:44:54,815 - script_runner - ERROR - 'NoneType' object is not iterable Traceback (most recent call last):
  File "C:\Users\ue71603\MG_Daten\github\badger\conv\gtfs_db_to_db.py", line 72, in main
    gtfs_db_to_db(source_database, Path(target_database_file), clean_database)
    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\conv\gtfs_db_to_db.py", line 46, in gtfs_db_to_db
    db_write.insert_any_object_on_queue(txn_write, gtfs_sj_processing(db_read, txn_read))
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\storage\mdbx\core\implementation.py", line 233, in insert_any_object_on_queue
    for obj in objects:
               ^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\transformers\gtfs.py", line 649, in gtfs_sj_processing
    for day_type_1, day_type_assignments_1, operating_periods_1 in query_daytype(db_read, txn, calendar_combinations):
                                                                   ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\transformers\gtfs.py", line 546, in query_daytype
    all_operating_periods.update(getIndex(operating_period))
                                 ~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\domain\netex\indexes\byid.py", line 9, in getIndex
    return {x.id: x for x in objects}
                             ^^^^^^^
TypeError: 'NoneType' object is not iterable

'NoneType' object is not iterable
2026-01-22 13:44:54,815 - script_runner - ERROR - 'NoneType' object is not iterable
Traceback (most recent call last):
  File "C:\Users\ue71603\MG_Daten\github\badger\tools\script_runner.py", line 440, in main
    result = load_and_run(script_name, script_args)
  File "C:\Users\ue71603\MG_Daten\github\badger\tools\script_runner.py", line 116, in load_and_run
    result = main_function(*args1)
  File "C:\Users\ue71603\MG_Daten\github\badger\conv\gtfs_db_to_db.py", line 75, in main
    raise e
  File "C:\Users\ue71603\MG_Daten\github\badger\conv\gtfs_db_to_db.py", line 72, in main
    gtfs_db_to_db(source_database, Path(target_database_file), clean_database)
    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\conv\gtfs_db_to_db.py", line 46, in gtfs_db_to_db
    db_write.insert_any_object_on_queue(txn_write, gtfs_sj_processing(db_read, txn_read))
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\storage\mdbx\core\implementation.py", line 233, in insert_any_object_on_queue
    for obj in objects:
               ^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\transformers\gtfs.py", line 649, in gtfs_sj_processing
    for day_type_1, day_type_assignments_1, operating_periods_1 in query_daytype(db_read, txn, calendar_combinations):
                                                                   ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\transformers\gtfs.py", line 546, in query_daytype
    all_operating_periods.update(getIndex(operating_period))
                                 ~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\badger\domain\netex\indexes\byid.py", line 9, in getIndex
    return {x.id: x for x in objects}
                             ^^^^^^^
TypeError: 'NoneType' object is not iterable
2026-01-22 13:44:54,818 - script_runner - INFO - Execution time: 0.3 seconds for nl4gtfs - step: 4: conv.gtfs_db_to_db C:/Users/ue71603/Downloads/nl4gtfs/03.mdbx C:/Users/ue71603/Downloads/nl4gtfs/04.mdbx

2026-01-22 13:44:54,818 - script_runner - ERROR - Script conv.gtfs_db_to_db returned an error. Terminating the block of scripts: nl4gtfs

```

--- a/tools/tool_scripts/script_regression_v1.1.txt
+++ b/tools/tool_scripts/script_regression_v1.1.txt
@@ -675,6 +675,22 @@
         {"script": "conv.gtfs_db_to_gtfs", "args": "%%dir%%/15-gtfs.mdbx %%dir%%/16-%%block%%-gtfs.zip"},
         {"script": "tools.gtfs_validator", "args": "%%dir%%/16-%%block%%-gtfs.zip %%dir%%"},
         {"script": "tools.gtfs_map_visualisation", "args": "%%dir%%/16-%%block%%-gtfs.zip %%dir%%/18-%%block%%-map.html 1"}
-         ]}
+         ]},
 
+         {
+    "block": "mmri_1a",
+    "url":"https://github.com/plannerstack/testset",
+    "download_url":"https://github.com/user-attachments/files/24860255/1a.zip",
+    "scripts": [
+        {"script": "clean_tmp", "args": "%%dir%%"},
+        {"script": "download_input_file", "args": "%%dir%%"},
+        {"script": "conv.gtfs_import_to_db", "args": "%%inputfilepath%% %%dir%%/03-gtfs-import.duckdb"},
+        {"script": "conv.gtfs_convert_to_db", "args": "%%dir%%/03-gtfs-import.duckdb %%dir%%/04-gtfs.mdbx"},
+        {"script": "conv.epip_db_to_db", "args": "%%dir%%/04-gtfs.mdbx %%dir%%/05-epip.mdbx"},
+        {"script": "conv.epip_db_to_xml", "args": "%%dir%%/05-epip.mdbx %%dir%%/06-%%block%%-netex.xml.gz"},
+        {"script": "conv.netex_to_db", "args": "[%%dir%%/06-%%block%%-netex.xml.gz] %%dir%%/07-epip.mdbx"},
+        {"script": "conv.gtfs_db_to_db", "args": "%%dir%%/07-epip.mdbx %%dir%%/08-gtfs.mdbx"},
+        {"script": "conv.gtfs_db_to_gtfs", "args": "%%dir%%/08-gtfs.mdbx %%dir%%/09-%%block%%-gtfs.zip"},
+        {"script": "tools.gtfs_validator", "args": "%%dir%%/09-%%block%%-gtfs.zip %%dir%%"}
+         ]}
 ]

--- a/transformers/epip.py
+++ b/transformers/epip.py
@@ -613,9 +613,9 @@ def epip_service_journey_generator(db_read: MdbxStorage, txn: TXN, generator_def
                 service_journey_pattern = service_journey_pattern_from_calls(sj, generator_defaults)
                 sj.journey_pattern_ref = getRef(service_journey_pattern)
 
-            # sj.passing_times = TimetabledPassingTimesRelStructure(
-            #     timetabled_passing_time=TimetablePassingTimesProfile.getTimetabledPassingtimesFromCalls(sj, service_journey_pattern)
-            # )
+            sj.passing_times = TimetabledPassingTimesRelStructure(
+                 timetabled_passing_time=TimetablePassingTimesProfile.getTimetabledPassingtimesFromCalls(sj, service_journey_pattern)
+            )
 
         elif sj.journey_pattern_ref and sj.time_demand_type_ref:
             service_journey_pattern: ServiceJourneyPattern = db_read.load_object_by_reference(txn, sj.journey_pattern_ref)

--- a/transformers/gtfs.py
+++ b/transformers/gtfs.py
@@ -446,7 +446,7 @@ def gtfs_sj_processing(db_read: MdbxStorage, txn: TXN) -> Generator[ServiceJourn
     yield from query_sj(db_read, txn)
 
     def query_daytype(db_read: MdbxStorage, txn: TXN, calendar_combinations):
-        service_calendars: List[ServiceCalendar] = list(db_read.iter_only_objects(txn, ServiceCalendar))
+        service_calendars: list[ServiceCalendar] = list(db_read.iter_only_objects(txn, ServiceCalendar))
 
         all_day_type_assignments = [
             x
@@ -586,7 +586,7 @@ def gtfs_sj_processing(db_read: MdbxStorage, txn: TXN) -> Generator[ServiceJourn
                         if operating_period is not None:
                             operating_period = [operating_period]
 
-                        all_operating_periods.update(getIndex(operating_period))
+                            all_operating_periods.update(getIndex(operating_period))
 
                         yield day_type, day_type_assignments, operating_period
                     else:

